lib/bitreserve/entities/base_entity.rb

```diff
@@ -3,15 +3,25 @@ module Entities
     class BaseEntity
       include Virtus.model
 
-      def self.from_collection(entities)
-        entities.map do |entity|
-          new(entity)
-        end
+      def self.from_collection(entities, content_range)
```

Comment:
@zamith that's a doc bug -- we only paginate large dataset endpoints. We'll fix it.